src/Misc/SettingsCloudflare.php

```diff
@@ -107,17 +107,13 @@ public function getSettingsGlobalData(): array
 						'component' => 'intro',
 						'introTitle' => \__('Geolocation', 'eightshift-forms'),
 						'introSubtitle' => \sprintf(\__("
-							By default geolocation uses cookie to store users location.
+							By default geolocation uses IP database to get users location.
```

Comment:
```suggestion
							<p>By default geolocation uses <a href='%1\$s' target='_blank' rel='noopener noreferrer'>GeoLite2</a> MaxMind ID database</a> to get users location.</p>
```

Reply:
no need for multiple links to the external resource we have this one on geolocaiton